snuba/datasets/processors/replays_processor.py

```diff
@@ -278,6 +282,37 @@ def process_replay_actions(
     ]
 
 
+def process_replay_event_link(
+    payload: Mapping[Any, Any],
+    processed: Mapping[Any, Any],
+    metadata: KafkaMessageMetadata,
+) -> dict[str, Any]:
+    event_id = None
+    event_severity = None
+    for level in LOG_LEVELS:
+        if payload.get(level + "_id") is not None:
+            event_id = to_uuid(payload[level + "_id"])
+            event_severity = level
+            break
+
+    if event_id is None or event_severity is None:
```

Comment:
How confident are you that this clause won't ever be hit? Even one bad event as you test this feature will shut down your consumer.

Reply:
Right, fair enough.